File: 762_Prime_Number_of_Set_Bits_in_Binary_Representation/solution.c
int count_ones(int n) {
    int count = 0;
    while (n > 0) { if (n % 2 == 1) count++; n /= 2; }
    return count;
}
bool is_prime(int n) {
    if (n < 2) return false;
    for (int i = 2; i * i <= n; i++) if (n % i == 0) return false;
    return true;
}
int countPrimeSetBits(int left, int right) {
    int count = 0;
    for (int i = left; i <= right; i++) if (is_prime(count_ones(i))) count++;
    return count;
}

File: 762_Prime_Number_of_Set_Bits_in_Binary_Representation/solution.py
class Solution:
    def countPrimeSetBits(self, left: int, right: int) -> int:
        def is_prime(n):
            if n < 2: return False
            for i in range(2, int(n**0.5) + 1):
                if n % i == 0: return False
            return True

        count = 0
        for i in range(left, right + 1):
            if is_prime(bin(i).count("1")):
                count += 1
        return count